OSMPBF/README.md

```diff
@@ -1,19 +1,48 @@
-*.proto files were downloaded from https://github.com/scrosby/OSM-binary/tree/master/src and changed in following ways:
+# Proto Files
 
-* To eliminate continuous conversions from `[]byte` to `string`, this
+*.proto files were downloaded from https://github.com/scrosby/OSM-binary/tree/master/src and changed in following ways:
 
+## Changes
+### StringTable
+- **File**: osmformat.proto
+- **Reason**: To eliminate continuous conversions from `[]byte` to `string`
+- **Old Code**:
 ```protobuf
 message StringTable {
    repeated bytes s = 1;
 }
 ```
-
-was changed to
+- **New Code**:
 
 ```protobuf
 message StringTable {
    repeated string s = 1;
 }
 ```
+- **Comptatibility**: This changes is expected to be fully compatible with all PBF files.
+
+### Optimization Goal
+- **File**: osmformat.proto, fileformat.proto
+- **Reason**: Better performance
+- **Added Code**:
+
+```protobuf
+option optimize_for = LITE_RUNTIME;
```

Comment:
It does not affect Go code generator: https://github.com/golang/protobuf/issues/788